pkg/autoscaler/stats_scraper.go

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package autoscaler
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/knative/serving/pkg/apis/serving"
+	"github.com/prometheus/common/expfmt"
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/labels"
+	corev1informers "k8s.io/client-go/informers/core/v1"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+)
+
+const (
+	trueInFloat64 = float64(1)
+	// TODO(yanweiguo): tuning the sample size
+	sampleSize          = 3
+	scraperTickInterval = time.Second
```

Comment:
Why not sample 1 pod every 1/3 of a second?  No need for artificial burstiness.

Reply:
Good point. Will do.